Dockerfile

```
@@ -0,0 +1,18 @@
+# set base image (host OS)
+FROM python:3.8-slim
+
+# set the working directory in the container
+WORKDIR /
```

Just a quick remark, `WORKDIR` should be the same in `Dockerfile` and `Dockerfile-dev` :grimacing: 

```suggestion
WORKDIR /usr/src/app
```

Or update `Dockerfile-dev` to have the following :point_down: 

```
WORKDIR /
```